# Calibration for Downward Camera with "down" Fisheye Lens
cameraMatrix:
  R0: # Row 0
    C0: 1017.435702078276
    C1: 0.0
    C2: 640.3416991556214
  R1: # Row 1
    C0: 0.0
    C1: 1019.41216344638
    C2: 492.993133999964
  R2: # Row 2
    C0: 0.0
    C1: 0.0
    C2: 1.0
distortionCoeffs:
  C0: -0.4437322504322723
  C1: 0.3428271653745262
  C2: -0.0004523911600893588
  C3: -0.001726899638527989
  C4: -0.2650600062547752
